libnmstate/nm/plugin.py

```diff
@@ -159,3 +175,21 @@ def _remove_ovs_bridge_unsupported_entries(iface_info):
     iface_info.pop(Interface.MTU, None)
 
     return iface_info
+
+
+def _nm_utils_decode_version(v=None):
+    if v is None:
+        v = NM.utils_version()
+    return (
+        str((v >> 16) & 0xFF)
+        + "."
+        + str((v >> 8) & 0xFF)
+        + "."
+        + str((v) & 0xFF)
+    )
+
+
+def _nm_client_decode_version(v=None):
+    if v is None:
+        v = NM.Client.get_version()
```

Comment:
You cannot do this. You need a client instance that you should get from `_check_version_mismatch()`. Then, you can do: `v = nm_client.get_version()`.

Reply:
Done, thank you!